rewrite.yml

```diff
@@ -0,0 +1,19 @@
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: CheckstyleAutoFixConfigured
```

Comment:
Let's get rid of `Configured` suffix and make the whole config consistent with the doc you wrote here - https://github.com/checkstyle/checkstyle-openrewrite-recipes?tab=readme-ov-file#setup. If any issues, please update the doc. This way we make sure our documentated instructions are correct.